projects/demo/src/modules/charts/line-chart/line-chart.template.html

```diff
@@ -1,5 +1,4 @@
 <tui-doc-page
-    i18n-header
```

Comment:
I need to delete it because otherwise it breaks
```ts
nativeElement.getAttribute('header') // undefined
```

Fortunately,, we dont need to translate header (because it is the name of the component).